runtime/compiler/p/codegen/PPCPrivateLinkage.cpp

```diff
@@ -1193,26 +1074,20 @@ void TR::PPCPrivateLinkage::createPrologue(TR::Instruction *cursor)
    if (intSavedFirst <= TR::RealRegister::LastGPR)
       {
       if (TR::Compiler->target.is64Bit() ||
-           !comp()->getOption(TR_DisableShrinkWrapping) ||
           (!comp()->getOption(TR_OptimizeForSpace) &&
            TR::RealRegister::LastGPR - intSavedFirst <= 3))
          {
-         // shrink wrapping
-         TR_BitVector *p = cg()->getPreservedRegsInPrologue();
          for (regIndex=intSavedFirst; regIndex<=TR::RealRegister::LastGPR; regIndex=(TR::RealRegister::RegNum)((uint32_t)regIndex+1))
             {
-            if (!p || p->get((uint32_t)regIndex))
-               cursor = generateMemSrc1Instruction(cg(),TR::InstOpCode::Op_st, firstNode, new (trHeapMemory()) TR::MemoryReference(stackPtr, argSize, TR::Compiler->om.sizeofReferenceAddress(), cg()), machine->getPPCRealRegister(regIndex), cursor);
+            cursor = generateMemSrc1Instruction(cg(),TR::InstOpCode::Op_st, firstNode, new (trHeapMemory()) TR::MemoryReference(stackPtr, argSize, TR::Compiler->om.sizeofReferenceAddress(), cg()), machine->getPPCRealRegister(regIndex), cursor);
+
```

Comment:
This comment is not related to the changes in this PR but more an observation I'll make about the code that's already there and want to document it somewhere.

The use of `om.sizeofReferenceAddress()` in this register preserving code seems wrong to me.  This code should be preserving the full register size for the target bitness rather than asking the object model what the size of a reference is.  As luck (design?) would have it, on 64-bit this always returns '8' and on 32-bit returns '4' regardless of whether compressed refs is enabled or not.

There are some curiosities that need to be sorted out here (like what should `sizeOfReferenceAddress` be returning from the object model, what breaks if that changes).  I'll open a separate issue for that investigation.

Reply:
@gita-omr FYI the above comment.

Reply:
Thanks. Will wait for the issue that @0xdaryl is going to open.